### ISSUE TITLE
Shortcut key for switching to Explorer sidebar

### ISSUE THREAD

**Issue by karlhorky** [CompuIves/codesandbox-client#1635]
# 🌈 Feature

I would like to configure a shortcut key for VS Code's showActiveFileInExplorer Action:

https://github.com/karlhorky/dotfiles/blob/99d27494eac1a755a930387dacf52569484f0234/settings/Library/Application%20Support/Code/User/keybindings.json#L67-L70

I would settle for a shortcut key to show the explorer. Is there anything like this or any way to do this easily while in VS Code mode?

**Comment by CompuIves** [comment]
Hey! Good request, it's already possible to open the explorer using ((Ctrl/CMD)+Shift+E) but I have yet to add the binding for `showActiveFileInExplorer`. I'll be focussing on these kinds of integrations over the next couple months.

**Comment by karlhorky** [comment]
> it's already possible to open the explorer using ((Ctrl/CMD)+Shift+E)

Ah very cool, cheers! Is this discoverable anywhere?

It would be cool to have a note next to the tooltip:

<img width="163" alt="Screen Shot 2019-03-21 at 14 46 49" src="https://user-images.githubusercontent.com/1935696/54756310-45612100-4be8-11e9-9f50-bf7f2b7f5388.png">

> I'll be focussing on these kinds of integrations over the next couple months.

Sounds great, thanks a bunch!

**Comment by CompuIves** [comment]
Good point, it's quite hidden right now:

![image](https://user-images.githubusercontent.com/587016/54756418-78a3b000-4be8-11e9-862e-ebe84cf59bf0.png)

I'll make sure to add that!

**Comment by karlhorky** [comment]
Cool, opened #1636 for this feature.

**Comment by lbogdan** [comment]
@karlhorky Is this sorted, then? If so, can you please close it? Thanks!

**Comment by karlhorky** [comment]
Nope, not yet. I've updated the title to match the outstanding TODO mentioned by @CompuIves in https://github.com/CompuIves/codesandbox-client/issues/1635#issuecomment-475233316

**Comment by karlhorky** [comment]
Update: Ctrl/CMD + Shift + E selects the current file when switching from another tab (like the Sandbox Info), but the shortcut does not open folders to show the current file if already on the Explorer tab.

**Comment by karlhorky** [comment]
not stale

**Comment by karlhorky** [comment]
i will fight to the death stalebot
